src/translate.ts

```diff
@@ -39,7 +39,7 @@ const translate = (language: Language) => ({
     const silent = languageKey ? language.silent || language.muted.indexOf(languageKey) !== -1 : false;
 
     // Default untranslated string, singular or plural.
-    const untranslated = defaultPlural && plurals.getTranslationIndex(languageKey, n) > 0 ? defaultPlural : msgid;
+    const untranslated = defaultPlural && n !== 1 ? defaultPlural : msgid;
```

Comment:
this change seems dangerous (and maybe is the reason the tests fail). why does it not rely on `getTranslationIndex` anymore?

Reply:
We have assumed the source code is written in English, so if we should display the untranslated message, why not use  English form rule(use plural form only if n!=1)?

Reply:
> We have assumed the source code is written in English

Ah, yes, I know that's gettext standard, but I myself and others I know of misuse gettext with different source languages. Please revert this just so we don't introduce any implicit breaking changes for now, then I can merge this! (assuming all the checks pass)

Reply:
added a new option `sourceCodeLanguage`, default undefined, if set, use it to get plural text.

Reply:
@lzurbriggen updated

Reply:
> added a new option sourceCodeLanguage, default undefined, if set, use it to get plural text.

I previously asked you to revert it because this is still a breaking change, but I will merge this for now an create a major version pre-release.